client-core/src/hd_seed.rs

```diff
@@ -9,7 +9,7 @@ use crate::types::AddressType;
 use crate::Mnemonic;
 
 /// Hierarchical Deterministic seed
-#[derive(Debug, Default, PartialEq, Decode, Encode)]
+#[derive(Debug, Clone, PartialEq, Decode, Encode)]
```

Comment:
The commit message is typo and mis-leading, I think the `Default` trait on `HDSeed` is insecure, the `Clone` instance is needed by `client-mock`.